cookiecutter.json

```diff
@@ -1,11 +1,12 @@
 {
-    "project_name": "project_name",
-    "repo_name": "{{ cookiecutter.project_name.lower().replace(' ', '_') }}",
-    "repo_owner": "",
-    "open_source_license": "Not open source",
-    "author_name": "",
-    "description": "My Project Description",
-    "base_docker_image": "manifoldai/orbyter-ml-dev:3.1",
-    "mlflow_uri": "/mnt/experiments",
-    "mlflow_artifact": ""
+  "project_name": "project_name",
+  "repo_name": "{{ cookiecutter.project_name.lower().replace(' ', '_') }}",
+  "module_name": "{{ cookiecutter.repo_name.lower().replace(' ', '_').replace('-', '_') }}",
```

Comment:
Since these are only specifying defaults for the user and not overriding their preferences, and I think it's assumed that the project name can have spaces or other special characters since they're only used in the docs, I think this is safe to do. The `pre_gen_project.py` check *does* fail if the user does not explicitly choose a valid module name, which in this case would actually require overriding the defaults.

Reply:
I see. We're allowing `data science notebooks` as a repo name -- but then we underscore it up appropriately. I guess I was proposing we put the onus on the user to type `data_science_notebooks` instead of using the spaces. What say you?

Reply:
Let's assume we are going to merge the meaning of `repo_name` and `module_name` for now, then really we're just allowing project names that can be whatever (e.g. "Orbyter AI"). So what this would do is let the default suggestion for the project/repo name be a valid one (e.g. `orbyer_ai`), which I think is just more user-friendly than providing a default suggestion that is invalid, and forcing the user to delete it.